stories/Atom/Global/Color/color.stories.mdx

```diff
@@ -0,0 +1,278 @@
+import { Meta, Story, ArgsTable } from '@storybook/addon-docs';
+import { Color } from './color'
+
+export const getCaptionForLocale = (locale) => {
+  switch(locale) {
+    case 'english': const engText = {PC: 'Primary colors', AC: 'Accent colors', BC: 'Balance of color palette', PT: 'Primary'}; return engText;
+    case 'arabic': const arabicText = {PC: 'الألوان الأساسية', AC: 'ألوان مميزة', BC: 'توازن لوحة الألوان', PT: 'ابتدائي'}; return arabicText;
+    case 'burmese': const burmeseText = {PC: 'မူလအရောင်များ', AC: 'အရောင်များကိုအလေးပေးပါ', BC: 'အရောင် palette ၏ Balance', PT: 'မူလတန်း'}; return burmeseText;
+    case 'japanese': const japaneseText = {PC: '原色', AC: 'アクセントカラー', BC: 'カラーパレットのバランス', PT: '主要な'}; return japaneseText;
+    default:
+      return {PC: 'Primary colors', AC: 'Accent colors', BC: 'Balance of color palette', PT: 'Primary'};
+  }
+};
+
+<Meta title="Atoms/Colors" component={Color} />
+
+<style>{`
+  div#story--atoms-colors--colors {
+    display: none;
+}
+  .subheading {
+    font-weight: 700;
+    font-size: 18px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  } 
+  .color_wrapper{
+    display: flex;
+  }
+  .color_item{
+    margin: 5px;
+  }
+  .color_box {
+    width: 90px;
+    height: 60px;
+    display: block;
+  }
+  .color_name{
+   font-size: 13px;
+   color: #666;
+   font-weight: 600;
+  margin: 10px 0;
+  }
+  .color_code{
+    font-size: 13px;
+    color: #666;
+  }
+
+  .p-blue-100 { background-color: #B5D5F5;}
+  .p-blue-200 {background-color: #94C4F5;}
+  .p-blue-300 { background-color: #6BABEB;}
+  .p-blue-400 {background-color: #4F95DD;}
+  .p-blue-500 {background-color: #3288CE;}
+  .p-blue-600 {background-color: #006EB5;}
+  .p-blue-700 {background-color: #1F5A95;}
+
+  .p-gray-100 { background-color: #FAFAFA;}
+  .p-gray-200 {background-color: #F7F7F7;}
+  .p-gray-300 { background-color: #EDEFF0;}
+  .p-gray-400 {background-color: #D4D6D8;}
+  .p-gray-500 {background-color: #A9B1B7;}
+  .p-gray-600 {background-color: #55606E;}
+  .p-gray-700 {background-color: #232E3D;}
+
+  .light-yellow { background-color: #FFE17E;}
+  .yellow {background-color: #FFEB00;}
+  .dark-yellow  { background-color: #FBC412;}
+  .light-red {background-color: #FFBCB7;}
+  .red {background-color: #EE402D;}
+  .dark-red {background-color: #D12800;}
+  .light-green {background-color: #B8ECB6;}
+  .green {background-color: #6DE354;}
+  .dark-green {background-color: #59BA47;}
+  .light-azure {background-color: #A2DAF3;}
+  .azure {background-color: #60D4F2;}
+  .dark-azure {background-color: #00C1FF;}
+
+`}</style>
+
+# Color Component
+Color component represent the all colors we are used into the website. we used all the colors as variables.
```

Comment:
please use the below text:

This component represents various colors that can be used in the website as per Brand guidelines. All these colors are defined as SCSS variables globally and accordingly used in all the components. The variables can be found in assets/scss/_variables.scss file. 

Colors are divided into two options - Primary and Accents colors.